src/aosm/azext_aosm/_configuration.py

```diff
@@ -57,7 +57,7 @@ def validate(self):
         Validate the configuration.
         """
         if not self.version:
-            raise ValidationError("version must be set.")
+            raise ValidationError("Version must be set.")
```

Comment:
Minor thing, but I've been trying to keep the error messages referencing fields exactly as they have to be supplied in input.json.  So I think that lowercase v is the right thing here.